test/PEX.spec.ts

```diff
@@ -233,6 +234,28 @@ describe('evaluate', () => {
     expect(proof.verificationMethod).toEqual('did:ethr:0x8D0E24509b79AfaB3A74Be1700ebF9769796B489#key');
   });
 
+  it("should throw error if proofOptions doesn't have a type", () => {
+    const pdSchema = getFile('./test/dif_pe_examples/pdV1/pd_driver_license_name.json');
```

Comment:
Shouldn't proof type be mandatory?

Reply:
yes, but we have a line of code for checking that

Reply:
If we have TypeScript to check that, why are we doing it?

Reply:
Huh what do you mean typescript is design time. Runtime is Javascript which has no type info at all

Reply:
I thought the code would be transpiled in a way that non null/undefined is enforced by the ? operator at runtime like it enforces undefined when a value is null or undefined in if statements. But that's not the case.